presto-main/src/main/java/io/prestosql/version/EmbedVersion.java

```diff
@@ -102,4 +141,19 @@ public Runnable embedVersion(Runnable runnable)
             throw new RuntimeException(throwable);
         }
     }
+
+    @Override
+    public <T> Callable<T> embedVersion(Callable<T> callable)
```

Comment:
move public above private stuff

Reply:
I added it below `public Runnable embedVersion(Runnable runnable)`

The private stuff above is for construction only, so IMO it belongs under the ctor.